setup.py

```diff
@@ -34,6 +35,9 @@ def read(*filenames, **kwargs):
             'flake8_quotes = flake8_quotes:QuoteChecker',
         ],
     },
+    license='MIT',
+    zip_safe=False,
```

Comment:
We aren't doing any compilation or reading files from disk, we should be good to label ourselves as `zip_safe=True`